OpenCO2_Sensor.ino

```diff
@@ -459,7 +459,7 @@ void goto_deep_sleep(int ms) {
   /* Wakeup by IO0 button */
   rtc_gpio_pullup_en(BUTTON);
   rtc_gpio_pulldown_dis(BUTTON);
-  esp_sleep_enable_ext1_wakeup_io((1ULL << BUTTON), ESP_EXT1_WAKEUP_ANY_LOW);
+  esp_sleep_enable_ext1_wakeup((1ULL << BUTTON), ESP_EXT1_WAKEUP_ANY_LOW);
```

Comment:
Thanks for proposing this change but I remember that with the latest esp32 release (3.1.3) I noticed that the _io version is needed.
https://docs.espressif.com/projects/esp-idf/en/v5.4/esp32s2/api-reference/system/sleep_modes.html#_CPPv428esp_sleep_enable_ext1_wakeup8uint64_t28esp_sleep_ext1_wakeup_mode_t
![image](https://github.com/user-attachments/assets/49502de4-fac7-4f0a-930d-7f679a0eb572)